samples/quickstart.js

```diff
@@ -17,7 +17,16 @@
 async function main() {
   // [START iot_quickstart]
   const iot = require('@google-cloud/iot');
-  const client = new iot.v1.DeviceManagerClient({fallback: 'rest'});
+  const region = 'us-central1';
+  const registry = 'ingressRegistry';
+  const projectId = 'ingressdevelopmentenv';
+  const client = new iot.v1.DeviceManagerClient(
+    null, //Will plan to remove this object
```

Comment:
Passing two nulls as the first two parameters seems odd. What do you think about turning all params into an options object and these two parameters would be optional properties? Like so
```
const client = new iot.v1.DeviceManagerClient({
    project,
    region,
    registry
})
```

Reply:
For this, I have removed those two parameters in the constructor, and with that I have removed all its related dependencies which are all google's auth lib or google gax lib. So for that, I have to comment out test cases as well because they follow google's constructor and we have changed the constructor initialization. So needs to comment it out for now and we can rewrite or rephrase it in the future. Let me know your thoughts on it.